app/Commands/EnableCommand.php

```diff
@@ -32,28 +38,90 @@ public function handle(Services $services): void
             return;
         }
 
-        $option = $this->menu('Services to enable:')->setTitleSeparator('=');
+        $option = $this->selectService();
 
-        foreach ($this->enableableServicesByCategory() as $category => $services) {
-            $menuItems = collect($services)->mapWithKeys(function ($service) {
-                return [$service['shortName'] => $service['name']];
-            })->toArray();
+        if (! $option) {
+            return;
+        }
+
+        $this->enable($option, $useDefaults);
+    }
+
+    private function selectService(): ?string
+    {
+        if ($this->environment->isWindowsOs()) {
+            return $this->windowsMenu();
+        }
 
+        return $this->defaultMenu();
+    }
+
+    private function defaultMenu(): ?string
+    {
+        $option = $this->menu(self::MENU_TITLE)->setTitleSeparator('=');
+
+        foreach ($this->enableableServicesByCategory() as $category => $services) {
             $separator = str_repeat('-', 1 + Str::length($category));
 
             $option->addStaticItem("{$category}:")
                 ->addStaticItem($separator)
-                ->addOptions($menuItems)
+                ->addOptions($this->menuItemsForServices($services))
                 ->addLineBreak('', 1);
         }
 
-        $option = $option->open();
+        return $option->open();
+    }
 
-        if (! $option) {
-            return;
+    private function windowsMenu($category = null): ?string
```

Comment:
My gut says we'll want to refactor this out, but since you've tied it up nicely into a single private method, that's fine for now--thanks!

Reply:
I think I can probably do something with this in a future PR.